src/app/shared/modules/selectors/area-selector/area-selector.module.ts

```diff
@@ -4,22 +4,14 @@ import { ReactiveFormsModule } from '@angular/forms';
 
 import { AreaSelectorModalComponent } from './area-selector-modal.component';
 import { AreaSelectorBtnComponent } from './area-selector-btn.component';
-import { highlightOptions } from '@keira-config/highlight.config';
 import { HighlightModule } from 'ngx-highlightjs';
 import { NgxDatatableModule } from '@swimlane/ngx-datatable';
 import { ModalModule } from 'ngx-bootstrap/modal';
 import { SearchButtonsModule } from '@keira-shared/modules/search-button/search-buttons.module';
 
 @NgModule({
   declarations: [AreaSelectorBtnComponent, AreaSelectorModalComponent],
-  imports: [
-    BrowserModule,
-    ReactiveFormsModule,
-    NgxDatatableModule,
-    ModalModule.forRoot(),
-    HighlightModule.forRoot(highlightOptions),
-    SearchButtonsModule,
-  ],
+  imports: [BrowserModule, ReactiveFormsModule, NgxDatatableModule, ModalModule.forRoot(), HighlightModule, SearchButtonsModule],
```

Comment:
`ModalModule.forRoot()` you should not call forRoot in a reusable module, this should only be called once, if you are importing area-selector.module in multiple places, you should remove `forRoot`